cmd/documentation/exchanges_templates/subscription.tmpl

```diff
@@ -0,0 +1,38 @@
+{{define "exchanges subscription" -}}
+{{template "header" .}}
+# Exchange Subscriptions
+
+Exchange Subscriptions are streams of data delivered via websocket.
+
+GoCryptoTrader engine will subscribe automatically to configured channels.
+A subset of exchanges currently support user configured channels, with the remaining using hardcoded defaults.
+See configuration Features.Subscriptions for whether an exchange is configurable.
+
+## Templating
+
+Exchange Contributors should implement `GetSubscriptionTemplate` to return a text/template Template.
+
+Exchanges are free to implement template caching, a map or a mono-template, inline or file templates.
+
+The template is provided with a single context structure:
+```go
+  S              *subscription.Subscription
+  AssetPairs     map[asset.Item]currency.Pairs
+  AssetSeparator string
+  PairSeparator  string
+```
+
+Subscriptions may fan out many channels for assets and pairs, to support exchanges which require individual subscriptions.  
+To allow the template to communicate how to handle it's output it should use the provided Separators:
```

Comment:
```suggestion
To allow the template to communicate how to handle it's output it should use the provided separators:
```

Reply:
Just spotted that `it's` is misspelt here as well.

Reply:
Fixed d20d247